src/model/types/index.ts

```diff
@@ -17,3 +17,48 @@ export type InvoiceData = {
 	memo?: string;
 	expiry?: number;
 };
+
+type RpcSubKinds = 'bolt11_mint_quote' | 'bolt11_melt_quote' | 'proof_state';
+
+export type RpcSubId = string | number | null;
+
+type JsonRpcParams = any;
+
+export type JsonRpcReqParams = {
+	kind: RpcSubKinds;
+	filters: Array<string>;
+	subId: string;
+};
+
+type JsonRpcSuccess<T = any> = {
+	jsonrpc: '2.0';
```

Comment:
maybe make jsonrpc `readonly`, to make sure the editor doesn't display the type as `string`